app/controllers/coronavirus_form/data_export_controller.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require "csv"
+
+class CoronavirusForm::DataExportController < ApplicationController
+  def show
+    respond_to do |format|
+      format.html { render controller_path }
+      format.csv { render csv: produce_csv(usage_statistics(params[:start_date], params[:end_date])), filename: "data-export" }
+    end
+  end
+
+  def usage_statistics(start_date, end_date)
+    questions = I18n.t("coronavirus_form.groups").map { |_, v| v[:questions] }.reduce(:merge)
+
+    start_date = Date.parse(sanitize(start_date)).beginning_of_day
+    end_date = Date.parse(sanitize(end_date)).end_of_day
+
+    results = {}
+    questions.each_key do |question|
+      question_text = questions.dig(question, :title)
+      counts = FormResponse
+        .where(created_at: start_date..end_date)
+        .select(Arel.sql(%{created_at::date, form_response -> '#{question}', COUNT(*)}))
```

Comment:
What do the `%` do?

Reply:
It creates a string (https://docs.ruby-lang.org/en/2.0.0/syntax/literals_rdoc.html#label-Percent+Strings).  Much like double quotes, but lets you include quotes in the string without needing to escape them.  I used this since an earlier iteration of the query included quote marks, but these can now be removed.